tests/helpers/highcharts.js

```diff
@@ -0,0 +1,15 @@
+import { findAll, waitUntil } from '@ember/test-helpers';
+
+export async function waitUntilAllChartsAreDrawn() {
```

Comment:
I wrote this test helper so that, when it's time to take a Percy snapshot, we know that all charts have been drawn. Otherwise, we may see something like the following:

<img width="800" alt="without_all_charts_settled" src="https://user-images.githubusercontent.com/16869656/106354069-708e1e80-62ef-11eb-8d61-83fb24ffc325.png">

Since `findAll` always returns an array, we can still use this test helper when there are 0 charts or only 1 chart in the test container.